openwillis/measures/video/face_landmark.py

```diff
@@ -251,7 +330,7 @@ def get_undected_markers(frame):
     df_landmark = pd.concat([df_common, df_coord], axis=1)
     return df_landmark
 
-def get_landmarks(path, error_info):
+def get_landmarks(path, error_info,bbox_list=[]):
```

Comment:
unused argument
```suggestion
def get_landmarks(path, bbox_list=[]):
```

Reply:
updated function signature in latest commit